desk/gembind.c

```diff
@@ -190,21 +72,34 @@ static LONG             addr_out[AO_SIZE];
 
 
 
-
-
-static WORD gem_if(WORD opcode)
+static __inline__ void gem(void)
 {
-    WORD i;
-    const BYTE *pctrl;
+	register long pb __asm__("d1") = (long)&gb;
+    __asm__ volatile (
+        "move.w  %1,d0\n\t"
+        "trap    #2\n\t"
+         :
+         : "r"(pb), "i"(200)
+         : "d0", "d2", "a0", "a1", "a2", "memory", "cc"
```

Comment:
What's exactly the difference? I don't see that used anywhere except in the Super() macro.